lms/djangoapps/lti_provider/management/commands/rescore_lti.py

```diff
@@ -0,0 +1,45 @@
+"""
+Management command to rescore all lti problems for the given course for all users.
+"""
+
+import textwrap
+
+from django.core.management import BaseCommand
+
+from opaque_keys.edx.keys import CourseKey
+
+from lti_provider.models import GradedAssignment
+from lti_provider import tasks
+from student.models import CourseEnrollment
+
+
+class Command(BaseCommand):
+    """
+    Rescore all lti problems for the given courses for all users.
+
+    Examples:
+
+        ./manage.py rescore_lti course-v1:edX+DemoX+Demo_Course course-v1:UBCx+course+2016-01
+    """
+
+    help = textwrap.dedent(__doc__)
+
+    def add_arguments(self, parser):
+        parser.add_argument(u'course_keys', type=CourseKey.from_string, nargs='+')
+
+    def handle(self, *args, **options):
+        for course_key in options[u'course_keys']:
+            for assignment in self._iter_lti_assignments(course_key):
+                tasks.send_composite_outcome.delay(
+                    assignment.user_id,
+                    unicode(course_key),
+                    assignment.id,
+                    assignment.version_number,
+                )
+
+    def _iter_lti_assignments(self, course_key):
+        """
+        Get all the graded assignments for the given course.
+        """
+
+        return GradedAssignment.objects.filter(course_key=course_key)
```

Comment:
I think this is handled by the `GradedAssignment` by design - `There will be only one row created for a given usage/consumer combination` (https://github.com/edx/edx-platform/blob/8f609d0251eb368984055e9069e702f100d204b3/lms/djangoapps/lti_provider/models.py#L103-L104), I just wanted to make doubly sure.